xtc/src/ledger.rs

```diff
@@ -194,10 +198,12 @@ async fn burn(args: BurnArguments) -> Result<TransactionId, BurnError> {
         Ok(()) => {
             let refunded = api::call::msg_cycles_refunded();
             let cycles = args.amount - refunded;
+            let new_fee = compute_fee(cycles);
```

Comment:
nice logic, I was just thinking to make the variable names a little bit more descriptive for example:
fee => fee_deducted
new_fee = actual_fee